### ISSUE TITLE
The Czeck language is a bit weird

### ISSUE THREAD

**Issue by LunaUrsa** [TripSit/combogen#15]
![image](https://user-images.githubusercontent.com/1836049/180030329-5f7aa03d-6d37-4d7a-bd7f-28c12b9eb7f8.png)

**Comment by ajar** [comment]
Will clean this up by simplifying the top part, basically? Our website isn't available in Czech anyway. Should we just remove that line and put something else?

**Comment by brozkeff** [comment]
Yes my patch to include the font ranges should fix this issue. For Czech, Slovak, Polish etc languages latin-ext should include all needed characters with diacritical marks, for Russian, Ukrainian, Bulgarian or Serbian the cyrillic should add the proper alphabet as well.

**Comment by MiranDaniel** [comment]
Is this still a problem?

**Comment by MiranDaniel** [comment]
Possible solution explained here: https://github.com/TripSit/combogen/pull/17#issuecomment-3464402082

I sadly cannot prove if it works or not since I don't have access to the Adobe account used for the font CDN.
